freeglut_joystick.c:1687: warning: the address of 'fgJoystick' will always evaluate as 'true'

--- freeglut-2.4.0/src/freeglut_joystick.c.orig	2005-05-04 14:53:48.000000000 +0200
+++ freeglut-2.4.0/src/freeglut_joystick.c	2006-11-01 21:59:56.032124750 +0100
@@ -1684,9 +1684,6 @@
 
   fgInitialiseJoysticks ();
 
-  if ( !fgJoystick )
-    return 0;
-
   if ( !fgState.JoysticksInitialised )
     return 0;
 
